DFC.ServiceTaxonomy.Editor/App_Data/Sites/Default/appsettings.template.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
This file is needed for provisioning the site in the release pipeline.

Reply:
Didn't remove it intentionally - will ensure it's re-added